simulate.cpp

```diff
@@ -12,6 +12,7 @@ class Simulator : public sf::RenderWindow {
     Simulator(void)
         : sf::RenderWindow(sf::VideoMode(800, 600), WINDOW_TITLE), maze() {
         calculateNodeSize();
+        maze = Maze::loadMazeFromFile("test.maze");
```

Comment:
I don't know exactly how the Visual Studio build works, but on Linux the `simulate` binary is placed in the root of the repository, and it makes the most sense to run from that directory. So it will fail to find this file.

This would be fixed by changing this to `"simulation/test.maze"` or moving the file to the root, but I presume that would break the current Visual Studio build.